templates/etc/quagga/Quagga.conf.j2

```diff
@@ -4,6 +4,7 @@ hostname {{ quagga_hostname }}
 password {{ quagga_password }}
 enable password {{ quagga_enable_password }}
 service password-encryption
+service integrated-vtysh-config
```

Comment:
It is what I've seen elsewhere. I guess it may not be needed since we'll be using ansible to make the config, not vtysh. I can make it depend on a boolean variable.